<launch>
    <arg name="target_frame" default="/base_link" />
    <arg name="pedestrian_array" default="/pedestrian_tracking/pedestrian_array" />
    <arg name="localisations" default="/pedestrian_localisation/localisations" />
    <arg name="marker" default="/pedestrian_localisation/marker_array" />
    
    <node pkg="strands_pedestrian_localisation" type="pedestrian_localisation" name="pedestrian_localisation" output="screen">
        <param name="target_frame" value="$(arg target_frame)" type="string"/>
        <param name="pedestrian_array" value="$(arg pedestrian_array)" type="string"/>
        <param name="localisations" value="$(arg localisations)" type="string"/>
        <param name="marker" value="$(arg marker)" type="string"/>
    </node>

</launch> 
